openpype/modules/deadline/plugins/publish/submit_fusion_deadline.py

```diff
@@ -7,22 +7,86 @@
 import pyblish.api
 
 from openpype.pipeline import legacy_io
-
-
-class FusionSubmitDeadline(pyblish.api.InstancePlugin):
+from openpype.pipeline.publish import (
+    OpenPypePyblishPluginMixin
+)
+from openpype.lib import (
+    BoolDef,
+    NumberDef
+)
+
+
+class FusionSubmitDeadline(
+    pyblish.api.InstancePlugin,
+    OpenPypePyblishPluginMixin
+):
     """Submit current Comp to Deadline
 
     Renders are submitted to a Deadline Web Service as
     supplied via settings key "DEADLINE_REST_URL".
 
     """
 
-    label = "Submit to Deadline"
+    label = "Submit Fusion to Deadline"
     order = pyblish.api.IntegratorOrder
     hosts = ["fusion"]
-    families = ["render.farm"]
+    families = ["render"]
+    targets = ["local"]
+
+    # presets
+    priority = 50
+    chunk_size = 1
+    concurrent_tasks = 1
+    group = ""
+    department = ""
+    limit_groups = {}
+    env_allowed_keys = []
+    env_search_replace_values = {}
+
+    @classmethod
+    def get_attribute_defs(cls):
+        return [
+            NumberDef(
+                "priority",
+                label="Priority",
+                default=cls.priority,
+                decimals=0
+            ),
+            NumberDef(
+                "chunk",
```

Comment:
this will be redone anyway so lets keep this as it is.

Reply:
> This will be redone anyway, so lets keep this as it is.

It's to be redone anyway. In what way? Backwards incompatible way? :) If we can I'd at least try to adopt the naming convention we want to use down the line.